DEI.md

```diff
@@ -0,0 +1,49 @@
+# Diversity, Equity, and Inclusion Project Statement
+
+<!---
+The DEI.md file was originally created in the CHAOSS project. This comment provides attribution of that work as defined under the MIT license. 
+-->
+
+<!---
+Please use the DEI.md Guide at https://github.com/AllInOpenSource/ProjectBadging/blob/main/Guide.DEI.md when creating your DEI.md file
+-->
+
+<img src="django-logo.png" alt="Django" style="width:200px;"/>
```

Comment:
Can we use SVG to look good at all resolutions and zoom levels?

Reply:
I'll see if I can find one 👍

Reply:
Updated ⭐